shared/config/config.go

```diff
@@ -15,6 +15,11 @@ type Config struct {
 	RootDir       string                 `json:"root_dir"`
 	GenesisSource *genesis.GenesisSource `json:"genesis_source"` // TECHDEBT(olshansky): we should be able to pass the struct in here.
 
+	// DISCUSS(deblasis): should we scope this into, let's say, RPCConfig?
+	RPCPort      string `json:"rpc_port"`       // TODO(deblasis): still unused, need to define default value, "8081" ?
```

Comment:
Let's stay consistent with tendermint for ports: https://docs.cosmos.network/master/core/grpc_rest.html

It'll be something people are used to.

Reply:
Ok, I'll use port 26657

Probably out of scope for this PR(s) but now a question I wanted to ask earlier too:

Should we offer GRPC + GRPC-Gateway APIs at some point in a Cosmos-like fashion?

I have done that in the past and as you can probably tell, I am a fan of protobufs and GRPC :)